Assignments/Program.cs

```diff
@@ -1,3 +1,192 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Academy23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// A12 - Wordle Game
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using static System.ConsoleColor;
+using System.Reflection;
+
+namespace Academy;
+public class A12 {
+   public static void Main () => new Wordle ().PlayTheGame ();
+}
+
+#region WordleGameClass----------------------------------------------------------------------------
+public class Wordle {
+   # region Properties --------------------------------------------------------
+   public string mSecretWord, mWord, mComment;
+   public bool mTesting = false, mGameOver;
+   int mX, mY;                                // // Reference X & Y coordinates
+   public int mTriesCount = 0;                // To count no.of tries made
+   public List<List<char>> mWords = new (6);  // List To store user entered letters
+   readonly Dictionary<char, ConsoleColor> mColorCode = new (); // To store foreground color to each char
+   public string[] mDictWords, mGuessWords;   // To store loaded strings
+   public StreamWriter testOutputFile = null; // Created for testing purpose

```

Comment:
Exposing all these fields is not the correct design. Think of a better way to do it. May be, in the testing mode some things like the secret word can be hardcoded etc.

Reply:
Is there any way to access private fields for outside the class, i.e. for testing purpose?